src/test/scala/beam/agentsim/agents/PersonAndTransitDriverSpec.scala

```diff
@@ -404,7 +404,8 @@ class PersonAndTransitDriverSpec
             )
           )
         ),
-        requestId = 1
+        requestId = 1,
+        request = None
```

Comment:
Rather than making all of these changes required should we have the default be `None`?

Reply:
I thought about this in the first place, but then it would make compiler happy and I could skip important places where I had to change

Reply:
I figured that might be the case...sgtm.